subprojects/diagnostics/src/main/java/org/gradle/api/reporting/components/ComponentReport.java

```diff
@@ -42,7 +41,6 @@
 /**
  * Displays some details about the software components produced by the project.
  */
-@Incubating
```

Comment:
We shouldn't promote software model types.

Reply:
Not promoting software model types any more.